controllers/elasticdl/elasticdljob_controller.go

```diff
@@ -75,7 +75,7 @@ type ElasticDLJobReconciler struct {
 // +kubebuilder:rbac:groups=training.kubedl.io,resources=elasticdljobs,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=training.kubedl.io,resources=elasticdljobs/status,verbs=get;update;patch
 
-func (r *ElasticDLJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+func (r *ElasticDLJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
```

Comment:
Better not change the interface

Reply:
this change is due to the change in controller-runtime. In controller-runtime@v0.8.3, `Reconciler interface` is changed into `Reconcile(context.Context, Request) (Result, error)`, added extra param `context`